docs/videos/healthcare-and-life-sciences/Factor-Xa-example/README.md

```diff
@@ -0,0 +1,43 @@
+# Factor Xa Example
+
+This example demonstrates a real life example of simulating the Factor Xa
+protein in water.
+
+It has been based on tutorials from Justin Lemhul (http://www.mdtutorials.com)
+and Alessandra Villa (https://tutorials.gromacs.org/).
```

Comment:
per https://creativecommons.org/licenses/by/4.0/, you may need to link to the license, e.g.:
This material is based, with changes, on the tutorials from [...] licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).

Please read more at the license location and make sure we do all all the required attribution, indication of changes, link to the license, etc.

Reply:
Done. I have updated the README to do the required attribution. I have also added the alternative licence to the submit.sh file. 

I believe this covers all of our bases. @vjb-collab, please LMK if we are going overboard here. We are basically treating this like a modified copy of the source material.

Reply:
looks good to me